### ISSUE TITLE
Create upstream v1.7.11

### ISSUE THREAD

**Issue by manolobevia** [cul-it/uls#1043]

**Comment by jgreidy** [comment]
Enabling Elementor Pro breaks the site. I suspect it is related to some 'hard deprications' in version v2.9.0.
I created a test environment here:
https://dashboard.pantheon.io/sites/0658f75c-a906-4cc2-9c57-44a8177e65e5#dev/backups/backup-log
The site is working, but once you activate Elementor Pro, it is just a white screen.
The backup from Mon, Jul 12th 2021 08:32:23 (UTC -04:00) has elementor pro deactivated.

Comparing the logs with and without elementor pro, I see this when enabled:

[12-Jul-2021 12:26:46 UTC] PHP Warning:  Cannot modify header information - headers already sent in /code/web/wp-config.php on line 244
[12-Jul-2021 12:26:46 UTC] PHP Warning:  Cannot modify header information - headers already sent in /code/web/wp-config.php on line 245

We need to figure out how to get Elementor Pro working, but when I tried to roll back to version v3.2.1 (in the upstream version v1.7.11), which is working on some sites, it still breaks.

I have to do a 2 week D&A sprint now.
